app/src/androidTest/java/ch/epfl/sdp/predator/PreyCatchFragmentTestActivityTest.kt

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.sdp.predator
+
+import androidx.test.core.app.launchActivity
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import org.junit.Assert
+import org.junit.Assert.assertNotEquals
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.ExpectedException
+import org.junit.runner.RunWith
+import java.lang.IllegalArgumentException
+import java.lang.NumberFormatException
+import java.lang.RuntimeException
+
+@RunWith(AndroidJUnit4::class)
+class PreyCatchFragmentTestActivityTest {
+
```

Comment:
Missing integration test for the interoperability between fragment and activity

Reply:
Really not sure how to do that :thinking: